### ISSUE TITLE
Using mangoapp results in black screen if no_display is used in .conf file

### ISSUE THREAD

**Issue by stele95** [flightlessmango/MangoHud#1283]
**Describe the bug**
When using mangoapp with the same config for mangohud that has ``no_display`` in it, it will show a black screen when mangoapp is initialised. Removing ``no_display`` from conf file starts mangoapp and image as expected.

**List relevant hardware/software information**
- Linux Distribution: EndeavourOS
- MangoHud version: latest git installed with mangohud-git from AUR

**To Reproduce**
1. Use [this](https://pastebin.com/hNmt5pPZ) config and uncomment ``no_display``
2. start ``gamescope -- sh -c "mangoapp& vkcube"`` from the terminal
3. Black screen appears as soon as mangoapp is initialised

**Expected behavior**
Mangoapp should ignore ``no_display`` if that is not supported for it

**Screenshots**
When using ``no_display``
![Screenshot_20240407_105704](https://github.com/flightlessmango/MangoHud/assets/26225081/532680ca-1039-4911-976a-8c82a708e82c)

After removing ``no_display``
![Screenshot_20240407_105715](https://github.com/flightlessmango/MangoHud/assets/26225081/d4d2566f-09f4-4502-ba9e-6e4762822d63)

**Comment by stele95** [comment]
Also, hidding mangoapp with the ``toggle_hud`` key results in a black screen if run with ``gamescope -- sh -c "mangoapp& vkcube"`` key and doesn't open it anymore

**Comment by gort818** [comment]
I believe we are waiting on a fix from gamescope for this.

**Comment by flightlessmango** [comment]
Yes we're waiting for @Joshua-Ashton to implement the `GAMESCOPE_NO_FOCUS` atom on the gamescope side

**Comment by flightlessmango** [comment]
This should be resolved

**Comment by stele95** [comment]
After testing with the latest git for MangoHud and Gamescope, I can confirm that it works as expected
